packages/host/app/components/operator-mode/overlays.gts

```diff
@@ -102,45 +105,43 @@ export default class OperatorModeOverlays extends Component<Signature> {
             />
-            {{#let (this.canWrite card) as |canWrite|}}
-              {{! Since there is just one item in the drop down, if that one item 
+            {{! Since there is just one item in the drop down, if that one item 
                   cannot be shown then we just don't show the drop down. This should 
                   change if we add more items in the dropdown }}
-              {{#if canWrite}}
-                <BoxelDropdown>
-                  <:trigger as |bindings|>
-                    <IconButton
-                      {{on
-                        'mouseenter'
-                        (fn this.setCurrentlyHoveredCard renderedCard)
-                      }}
-                      {{on 'mouseleave' (fn this.setCurrentlyHoveredCard null)}}
-                      class='hover-button more-actions'
-                      @icon={{ThreeDotsHorizontal}}
-                      aria-label='more actions'
-                      {{bindings}}
-                    />
-                  </:trigger>
-                  <:content as |dd|>
-                    <Menu
-                      @closeMenu={{dd.close}}
-                      @items={{array
-                        (menuItem
-                          'Delete'
-                          (fn @publicAPI.delete card)
-                          icon=IconTrash
-                          dangerous=true
-                        )
-                      }}
-                      {{on
-                        'mouseenter'
-                        (fn this.setCurrentlyHoveredCard renderedCard)
-                      }}
-                    />
-                  </:content>
-                </BoxelDropdown>
-              {{/if}}
-            {{/let}}
+            {{#if canWrite}}
+              <BoxelDropdown>
+                <:trigger as |bindings|>
+                  <IconButton
+                    {{on
+                      'mouseenter'
+                      (fn this.setCurrentlyHoveredCard renderedCard)
+                    }}
+                    {{on 'mouseleave' (fn this.setCurrentlyHoveredCard null)}}
+                    class='hover-button more-actions'
+                    @icon={{ThreeDotsHorizontal}}
+                    aria-label='more actions'
+                    {{bindings}}
+                  />
+                </:trigger>
+                <:content as |dd|>
+                  <Menu
+                    @closeMenu={{dd.close}}
+                    @items={{array
+                      (menuItem
+                        'Delete'
+                        (fn @publicAPI.delete card)
+                        icon=IconTrash
+                        dangerous=true
+                      )
+                    }}
+                    {{on
+                      'mouseenter'
+                      (fn this.setCurrentlyHoveredCard renderedCard)
+                    }}
+                  />
+                </:content>
+              </BoxelDropdown>
+            {{/if}}
```

Comment:
only change here was removal of the wrapping `{{#let}}...{{/let}}` and indenting